ahmed-yara-sara/src/components/Search.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component, useEffect ,useState } from "react";
+import "react-bootstrap";
+import { Spinner, Form, FormControl, Button } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+import DropdownMenu from "./Dropdown"
+export default function Search(props) {
+  const [isSubmitted, submitForm] = React.useState(false);
```

Comment:
make this state like this
```suggestion
  const [isSubmitted, setIsSubmitted] = React.useState(false);
```

I think it is more clean now
what do you think?